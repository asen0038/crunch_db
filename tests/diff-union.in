set a 1 2 4
set b 2 9 0
set c
diff a b c
inter a b c
union a b c
list entries
bye
