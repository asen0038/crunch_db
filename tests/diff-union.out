>
ok

>
ok

>
ok

>
[0 1 4 9]

>
[]

>
[0 1 2 4 9]

>
c []
b [2 9 0]
a [1 2 4]

>
bye
